packages/react-components/src/molecules/GroupTools.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import css from '@emotion/css';
+import { GroupResponse } from '@asap-hub/model';
+
+import { Card, Headline3, Paragraph, Link } from '../atoms';
+import { getIconFromUrl } from '../utils';
+import { perRem } from '../pixels';
+
+const BUTTON_SPACING = 28 / perRem;
+const buttons = css({
+  display: 'flex',
+  flexFlow: 'wrap',
+  marginTop: `${18 / perRem}em`,
+  width: `calc(100% + ${BUTTON_SPACING}em)`,
+  marginRight: `-${BUTTON_SPACING}em`,
```

Comment:
y tho?

Reply:
I don't see another way to get the spacing in the design.

Reply:
ok I found out